pkg/csi/service/wcp/controller.go

```diff
@@ -840,7 +840,7 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		FilterSuspendedDatastores:      filterSuspendedDatastores,
 		UseSupervisorId:                isTKGSHAEnabled,
 		IsByokEnabled:                  isByokEnabled,
-		IsCSITransactionSupportEnabled: isCSITransactionSupportEnabled,
+		IsCSITransactionSupportEnabled: isFCDTransactionSupportEnabled,
```

Comment:
If CSI Transaction support is dependent on FCD Transaction support in WCP, then for Vanilla CSI Transaction support is not dependent on FCD Transaction support ? Is that the correct understanding

Reply:
For vanilla, we check both VC version and FSS from configmap